docs/reference/transform/setup.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[transform-setup]]
+=== Set up {transforms}
+++++
+<titleabbrev>Setup</titleabbrev>
+++++
+
+To use the {transforms}, you must have the
+{subscriptions}[appropriate license] and at least one
+<<transform-setup-nodes,{transform} node>> in your {es} cluster. If {stack}
+{security-features} are enabled, you must also ensure your users have the
+<<transform-privileges,necessary privileges>>.
+
+[discrete]
+[[transform-setup-nodes]]
+==== {transform-cap} nodes
+
+To use {transforms}, there must be at least one node in your cluster with
+`xpack.transform.enabled` and `node.transform` set to `true`. All data nodes and
+master-eligible nodes must have `xpack.transform.enabled` set to `true`, which
+is the default behavior.
+
+You can limit which nodes run resource-intensive activity related to
```

Comment:
I would scratch `resource-intensive`, most transform load is caused by search/aggregations which happens where the data sits. The transform task is not resource intensive, it's a light-weight coordinating task. Maybe:

You can limit which nodes run the coordinating transform task by ...